app/controllers/users.js

```diff
@@ -35,7 +35,8 @@ class UsersController extends DefaultController {
       const settings = _.get(req.User, key);
       res.json(settings);
     } else {
-      res.status(404).json({error: `No settings for ${namespace}`});
+      // no settings stored under that namespace - give empty object
+      res.json({});
```

Comment:
As CircleCI promptly notified, this change does break a few api tests.

Reply:
yep - test's covers this case :) - should be easy test change.. If you have time to finalise this...